package.json

```
@@ -1,28 +1,54 @@
 {
   "name": "single-spa-angular",
-  "version": "3.3.0",
-  "description": "Helpers for building single-spa applications which use Angular 2",
-  "main": "lib/browser-lib/single-spa-angular.js",
-  "schematics": "./schematics.json",
+  "version": "0.0.0",
+  "description": "Helpers for building single-spa applications which use Angular 2+",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/single-spa/single-spa-angular.git"
+  },
+  "keywords": [
+    "single-spa",
+    "angular"
+  ],
+  "author": "Joel Denning",
+  "maintainers": [
+    "Joel Denning",
+    "Artur Androsovych"
+  ],
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/single-spa/single-spa-angular/issues"
+  },
+  "homepage": "https://github.com/single-spa/single-spa-angular#readme",
+  "private": true,
   "scripts": {
     "prepublishOnly": "yarn build && yarn test",
-    "build": "yarn clean && yarn build:webpack && yarn parcel && yarn spa && yarn :copy-schematic-files",
-    "clean": "rimraf lib parcel",
-    "test": "jest",
+    "clean": "rimraf lib pacel",
+    "build": "run-s clean build:single-spa-angular build:webpack build:schematics",
+    "test": "yarn clean && yarn jest",
     "lint": "eslint ./src/**/*.ts",
     ":copy-schematic-files": "cpx \"src/**/_files/**/**\" lib",
     "parcel": "ngc -p tsconfig.parcel.json",
-    "spa": "tsc -p tsconfig.spa.json",
-    "build:webpack": "tsc -p tsconfig.node.json",
+    "// - SINGLE-SPA-ANGULAR": "Scripts for building single-spa-angular",
+    "build:single-spa-angular": "yarn ts-node tools/build",
+    "// - WEBPACK": "Scripts for building Webpack config transformer",
+    "prebuild:webpack": "rimraf lib/lib",
+    "build:webpack": "tsc -p tsconfig.webpack.json",
+    "// - SCHEMATICS": "Scripts for building schematics and copying its files",
+    "prebuild:schematics": "rimraf lib/schematics",
+    "build:schematics": "tsc -p tsconfig.schematics.json && run-s copy:schematics:*",
+    "copy:schematics:json": "cpx schematics/schematics.json lib/schematics",
+    "copy:schematics:schema": "cpx \"schematics/ng-add/schema*\" lib/schematics/ng-add",
+    "copy:schematics:files": "cpx \"schematics/ng-add/_files/**/**\" lib/schematics/ng-add/_files",
     "format": "prettier './**/*' --write",
     "// - INTEGRATION INSTALLS": "Install packages for integration apps",
     "install:integration:shop": "yarn --cwd integration/shop install",
     "install:integration:portal": "yarn --cwd integration/portal install",
-    "install:integration": "yarn install:integration:shop && yarn install:integration:portal",
+    "install:integration": "run-s install:integration:shop install:integration:portal",
```

Why sequential? Couldn't they run concurrently?

```suggestion
    "install:integration": "concurrently -n w: 'yarn:install:integration:*'",
```